### PR TITLE
Adding support for setting custom json attributes in first_boot.json

### PR DESCRIPTION
**Pull request by Vasu1105**
w.r.t https://github.com/chef-partners/azure-chef-extension/issues/164

### CODE REVIEW

File: lib/chef/azure/core/bootstrap_context.rb [hunk 1]
```diff
@@ -21,6 +21,10 @@ def chef_server_ssl_cert
           @chef_config[:chef_server_ssl_cert_content]
         end
 
+        def first_boot
+          first_boot_attributes_and_run_list = (@config[:first_boot_attributes] || {}).merge(:run_list => @run_list)
```
Comment (btm) [inline lib/chef/azure/core/bootstrap_context.rb]: You don't use the variable so you can drop the assignment here and just have `(@config[:first_boot_attributes] || {}).merge(:run_list => @run_list)`

A simple version would be `Hash(@config[:first_boot_attributes]).merge(:run_list => @run_list)` as Hash with either return the argument passed as a hash, which it already is, or an empty hash if the argument passed evaluates to nil.

File: lib/chef/azure/commands/enable.rb [hunk 2]
```diff
@@ -173,6 +173,7 @@ def configure_chef_only_once
         config[:node_ssl_verify_mode] =  bootstrap_options['node_ssl_verify_mode'] if bootstrap_options['node_ssl_verify_mode']
         runlist = @run_list.empty? ? [] : escape_runlist(@run_list)
         load_cloud_attributes_in_hints if ! @ohai_hints.empty?
+        config[:first_boot_attributes] = eval(@first_boot_attributes)
```
Comment (btm) [inline lib/chef/azure/commands/enable.rb]: Why `eval`? Is there ruby code in this variable? It seems unlikely. If there is, we need more clarity on what goes into this variable.

Comment (Vasu1105) [inline lib/chef/azure/commands/enable.rb]: @btm here eval is used to convert json string into Hash the the @first_boot_attributes has contains string e.g. "{\"name\"=>\"testjson\"}" to convert this in {"name"=>"testson"} eval is used which is actually a dirty way. I am checking if there is anything better to do this.

Comment (Vasu1105) [inline lib/chef/azure/commands/enable.rb]: @btm Fixed this. Ready for review

**Pull request merged**